src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -75,11 +75,11 @@
 
 (defn system-message-contact-request
   [{:keys [chat-id timestamp-str from]} type]
-  (let [[primary-name _]     (rf/sub [:contacts/contact-two-names-by-identity chat-id])
-        contact              (rf/sub [:contacts/contact-by-address chat-id])
-        photo-path           (when (seq (:images contact)) (rf/sub [:chats/photo-path chat-id]))
-        customization-color  (rf/sub [:profile/customization-color])
-        {:keys [public-key]} (rf/sub [:profile/profile])]
+  (let [[primary-name _]    (rf/sub [:contacts/contact-two-names-by-identity chat-id])
+        contact             (rf/sub [:contacts/contact-by-address chat-id])
+        photo-path          (when (seq (:images contact)) (rf/sub [:chats/photo-path chat-id]))
+        customization-color (rf/sub [:profile/customization-color])
+        public-key          (rf/sub [:multiaccount/public-key])]
```

Comment:
I thought we moved away from :multiaccount,  right @flexsurfer ?

Reply:
i was meaning to ask why some subs in the same ns are called both profile and multiaccount. `:multiaccount/public-key` is still heavily used though.

Reply:
cc @alwx

Reply:
idk probably leftovers, we shouldn't use `:multiaccount` any longer

Reply:
yes, it should be a separate refactoring

Reply:
Because there's no need to subscribe to the entire profile when we only need the `public-key` from it. We use a layer-3 subscription instead to avoid re-renders when something unrelated changes in `:profile/profile`.

I can add a `:profile/public-key` sub or can leave it until the `:multiaccount` one is renamed, either works for me.

Reply:
as you're changing this i would say better to do it properly then, pls add `:profile/public-key` sub, thank you

Reply:
done. Also added a small comment that `:multiaccount/public-key` is deprecated.